tests/userTests.py

```diff
@@ -63,7 +63,7 @@ def test_registration(self):
         self.logout()
         email = self.test_users["change_user_email"]
         self.setUpToken(email)
-        postJson = {"email": email, "name": "user", "agency": "agency", "title": "title", "password": self.user_password}
+        postJson = {"email": email, "name": "user", "cgac_code": "agency", "title": "title", "password": self.user_password}
```

Comment:
Same comment as above re: making the fake JSON explicit about what type of info is being represented.